DIAGRAM.md

```diff
@@ -16,16 +20,25 @@ flowchart BT
   subgraph band_mgmt [ ]
     direction LR
 
-    pa[Principal Architect]
     em[Engineering Manager]
-    cem[Head of cloud and security engineering]
     qam[QA/QE Manager]
     eol[Engineering Operations Lead]
   end
 
   subgraph band_teams [ ]
     direction LR
 
+    subgraph report_cto [ ]
+      direction BT
+      
+      pa[Principal Architect]
+      a[Architect]
+      de[Distinguised Engineer]
+
+      de -.-> a
```

Comment:
@andrew-brown-lm did you want to remove this line?
Or else do some fancier rearrangement of this subgraph?

Reply:
Removed line